CHANGELOG.md

```diff
@@ -1,44 +1,15 @@
 # Changelog
-All notable changes to this project 6.5.x per each release will be documented in this file.
+All notable changes to this project `7.0.x` per each release will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [v6.5.0.3] - 2018-08-28
+## [v7.0.0.1] - TBA
```

Comment:
@Aaquiff can you set the release date and commit it to the `7.0.x` branch via a separate PR?

Reply:
Fixed in #162